### PR TITLE
Extend the blame_verifier_report_plot to plot paper configs with multiple projects

### PR DESCRIPTION
**Pull request by cormensratio**
Extends the blame_verifier_report_plot to plot paper configs with multiple projects
closes: se-passau/VaRA#657

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- varats/varats/plots/blame_verifier_report_plots.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6260905&bid=20386812)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- varats/varats/plots/blame_verifier_report_plots.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6260905&bid=20386812)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- varats/varats/plots/blame_verifier_report_plots.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6260905&bid=20386812)

File: varats/varats/plots/blame_verifier_report_plots.py [hunk 1]
```diff
@@ -2,130 +2,248 @@
...
 import logging
 import typing as tp
+from pathlib import Path
 
 import matplotlib.pyplot as plt
 import matplotlib.style as style
 import matplotlib.ticker as mtick
 import numpy as np
 import pandas as pd
+from sklearn import preprocessing
```
Comment (vulder) [inline varats/varats/plots/blame_verifier_report_plots.py:12]: When you use this for the plot you need to update the dependencies in `setup.py` and requirements.txt

File: varats/varats/plots/blame_verifier_report_plots.py [hunk 2]
```diff
@@ -2,130 +2,248 @@
...
-        }
-        if extra_plot_cfg is not None:
-            plot_cfg.update(extra_plot_cfg)
+def _is_multi_cs_plot() -> bool:
+    if len(PC.get_paper_config().get_all_case_studies()) > 1:
+        return True
 
-        style.use(self.style)
-        verifier_plot_df = self._get_verifier_data()
+    return False
```
Comment (vulder) [inline varats/varats/plots/blame_verifier_report_plots.py:141]: You can keep this for now but we should change this after wards so the user can specify from the outside if he wants to see all projects or a specific one, even when there is more than one case study in the current PC.

**Review by vulder — CHANGES_REQUESTED** [review]
overall this looks good for a first version of the plot

File: varats/varats/plots/blame_verifier_report_plots.py (hunk 1 above)
Comment (cormensratio) [reply varats/varats/plots/blame_verifier_report_plots.py:12]: I think that it is already included in both files as scikit-learn. Is there a way to efficiently search which imported python module comes from what package?

File: varats/varats/plots/blame_verifier_report_plots.py (hunk 2 above)
Comment (cormensratio) [reply varats/varats/plots/blame_verifier_report_plots.py:141]: Okay, just tell me when you want it to be changed.

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- varats/varats/plots/blame_verifier_report_plots.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/se-passau/VaRA-Tool-Suite/pullRequest?prid=6260905&bid=20386812)

File: varats/varats/plots/blame_verifier_report_plots.py (hunk 1 above)
Comment (vulder) [reply varats/varats/plots/blame_verifier_report_plots.py:12]: Ah, kk I miss searched the lib name and did not find in the setup.py. Should be alright :)

**Review by vulder — APPROVED** [review]

**Pull request merged**